src/docs/integrations/github.mdx

```diff
@@ -7,7 +7,7 @@ title: "GitHub Integration"
 To configure the GitHub integration you'll need to create a GitHub app and obtain credentials.
 
 <Alert level="warning">
-  The GitHub App name must not contain any spaces.
+  The GitHub Org Slug must not contain any spaces.
```

Comment:
Github Application Name **may** have spaces. And that's not important on the form below this alert box. What's important is the Github organization slug.